content/en/docs/concepts/configuration/scheduler-perf-tuning.md

```diff
@@ -8,31 +8,37 @@ weight: 70
 
 {{% capture overview %}}
 
-{{< feature-state for_k8s_version="1.12" >}}
+{{< feature-state for_k8s_version="1.14" state="beta" >}}
 
 Kube-scheduler is the Kubernetes default scheduler. It is responsible for
 placement of Pods on Nodes in a cluster. Nodes in a cluster that meet the
 scheduling requirements of a Pod are called "feasible" Nodes for the Pod. The
 scheduler finds feasible Nodes for a Pod and then runs a set of functions to
 score the feasible Nodes and picks a Node with the highest score among the
-feasible ones to run the Pod. The scheduler then notifies the API server about this
-decision in a process called "Binding".
+feasible ones to run the Pod. The scheduler then notifies the API server about
+this decision in a process called "Binding".
 
 {{% /capture %}}
 
 {{% capture body %}}
 
 ## Percentage of Nodes to Score
 
-Before Kubernetes 1.12, Kube-scheduler used to check the feasibility of all the
-nodes in a cluster and then scored the feasible ones. Kubernetes 1.12 has a new
-feature that allows the scheduler to stop looking for more feasible nodes once
-it finds a certain number of them. This improves the scheduler's performance in
-large clusters. The number is specified as a percentage of the cluster size and
-is controlled by a configuration option called `percentageOfNodesToScore`. The
-range should be between 1 and 100. Other values are considered as 100%. The
-default value of this option is 50%. A cluster administrator can change this value by providing a
-different value in the scheduler configuration. However, it may not be necessary to change this value.
+Before Kubernetes 1.12, Kube-scheduler used to check the feasibility of all
+nodes in a cluster and then scored the feasible ones. Kubernetes 1.12 added a
+new feature that allows the scheduler to stop looking for more feasible nodes
+once it finds a certain number of them. This improves the scheduler's
+performance in large clusters. The number is specified as a percentage of the
+cluster size. The percentage can be controlled by a configuration option called
+`percentageOfNodesToScore`. The range should be between 1 and 100. Larger values
+are considered as 100%. Kubernetes 1.14 has logic to find the percentage of
```

Comment:
Is the linear scaling disabled when a user specifies an explicit value for the flag? How is that represented in config/flags? i.e. what is the value of `percentageOfNodesToScore` if I run `kube-scheduler --write-config-to=/dev/stdout`?

EDIT: I answered my own question and the answer is `0`. If `0` means auto-scale, then we should document that explicitly.

Reply:
Good point. Added.